src/authorizations/components/TokensTab.tsx

```diff
@@ -319,7 +323,9 @@ class TokensTab extends PureComponent<Props, State> {
 
     if (isEmpty(tokensSelectedForBatchOperation)) {
       this.setState({tokensSelectedForBatchOperation: tokensOnCurrentPage})
+      event(`bulkAction.selectAll`, {count: tokensOnCurrentPage.length})
     } else {
+      event(`bulkAction.deSelectAll`)
```

Comment:
should these be:

```ts
'bulkAction.tokens.selectAll'
'bulkAction.tokens.deselectAll'
```
so if we add more bulk actions, we know it's for tokens?

Also, no need to use the template string (\`) if there's not going to be interpolation (`${variable}`)